app/js/components/orderEntry/SelectOrderType.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as orderTypes from './orderTypes';
+
+const btnStyle = {
```

Comment:
If you could avoid inline styling, I think it is a good practice since the code will be huge at some point.

Reply:
Thaks @hadijahkyampeire 
I will move the styles into a CSS file.